proposals/dark_mode_extension_icons.md

```diff
@@ -200,7 +200,8 @@ starting with smaller sizes if available, retreating to the nearest larger size.
 1. The `"16"` is a size in `{"16": "icon.png"}` and any number can be used as a
 size, per the browser’s icon requirements. The word `"any"` can also be used in
 place of a number to represent an icon that can be shown at any size (usually
-vector images). The icon size used by the browser will be determined as follows:
+vector images). Does that mean that "any" will only have an svg and not a png?
```

Comment:
```suggestion
vector images).
```

As currently with the `manifest.icons` key, you can already specify icons in a size not matching the size specified, I do not see a reason to not support the `any` keyword for raster images.